coap-gateway/service/config.go

```diff
@@ -7,6 +7,15 @@ import (
 	"github.com/plgd-dev/kit/sync/task/queue"
 )
 
+type DeviceStatusValidityConfig struct {
```

Comment:
There is still pending MR from @Trishia . Is this going to create a conflict? Shouldn't we firstly merge Trishias changes?

Reply:
There is lot's of work to finish it - mainly in tests. So I don't want to wait for it. At first @ondrejtomcik pls validate yaml files and than when you are ok with them I can fix tests there.